### ISSUE TITLE
Undocumented: uniprop, unimatch

### ISSUE THREAD

**Issue by zoffixznet** [perl6/doc#900]
There are [some routines](https://github.com/rakudo/rakudo/blob/20ed9e2/src/core/Cool.pm#L107) that don't appear to be documented (nothing shows up in the search):

``` perl6
    method uniprop(|c)      { uniprop(self, |c) }
    method unimatch(|c)     { unimatch(self, |c) }

    method uniprop-int(|c)  { uniprop-int(self, |c) }
    method uniprop-bool(|c) { uniprop-bool(self, |c) }
    method uniprop-str(|c)  { uniprop-str(self, |c) }
```

The last three are _not_ in the spectest and I don't know whether they should be.

**Comment by samcv** [comment]
I've added uniprop, unimatch and uniprops to the docs. I also think uniprop-str/bool/int are kind of redundant... A better part of the spec would be that uniprop would return Int's, Bool's, Str's depending on the property.

**Comment by samcv** [comment]
This has been fixed in a4292f52
Closing 👍
